bciers/apps/compliance/src/app/components/compliance-summaries/ComplianceSummariesPage.tsx

```diff
@@ -1,13 +1,24 @@
 import { DataGridSearchParams } from "@/compliance/src/app/types";
 import { fetchComplianceSummariesPageData } from "@/compliance/src/app/utils/fetchComplianceSummariesPageData";
 import ComplianceSummariesDataGrid from "@/compliance/src/app/components/compliance-summaries/ComplianceSummariesDataGrid";
+import { getSessionRole } from "@bciers/utils/src/sessionUtils";
+import getReportVersionIDsWithActionedECs from "@/compliance/src/app/utils/getReportVersionIDsWithActionedECs";
 
 export default async function ComplianceSummariesPage({
   searchParams,
 }: Readonly<{
   searchParams: DataGridSearchParams;
 }>) {
   const initialData = await fetchComplianceSummariesPageData(searchParams);
+  const frontEndRole = await getSessionRole();
+  const isCasStaff = frontEndRole.startsWith("cas_");
+  const actionedECReportVersionIDs = await getReportVersionIDsWithActionedECs();
```

Comment:
I suggest we avoid creating a separate endpoint just to fetch all IDs for EC status checks. Instead, we can add a new field to the `ComplianceReportVersionListOut` schema that includes the EC status. Then, we can simply check that status in the ActionCell. This approach eliminates the need for a separate endpoint, along with its associated tests and service logic.